docker.yaml

```diff
@@ -127,6 +127,16 @@ subpackages:
       - runs: |
           install -Dm755 /home/docker-library/dockerd-entrypoint.sh "${{targets.subpkgdir}}"/usr/bin/dockerd-entrypoint.sh
 
+  - name: dockerd-rootless
+    description: "dockerd rootless"
+    dependencies:
+      runtime:
+        - rootlesskit
+    pipeline:
+      - runs: |
+          install -Dm755 contrib/dockerd-rootless-setuptool.sh "${{targets.subpkgdir}}"/usr/bin/dockerd-rootless-setuptool.sh
+          install -Dm755 contrib/dockerd-rootless.sh "${{targets.subpkgdir}}"/usr/bin/dockerd-rootless.sh
```

Comment:
we may change it as `/usr/bin/dockerd-rootless` instead of ~/usr/bin/dockerd-rootless.sh~ according to here https://git.alpinelinux.org/aports/tree/community/docker/APKBUILD#n192